04.ls/ls.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ORDER_COLUMN = 3
+BLANK = 2
+
+def main
+  files = Dir.glob('*')
+
+  remainder = files.size % ORDER_COLUMN
+  padding_count = (ORDER_COLUMN - remainder) % ORDER_COLUMN
+  paddings = [''] * padding_count
+  padded_files = files + paddings
+  rows_count = padded_files.size / ORDER_COLUMN
+  files_grid = padded_files.each_slice(rows_count).to_a
+  display_files = files_grid.transpose
+  column_width = files.map(&:length).max
+  output(display_files, column_width)
+end
+
+def output(display_files, column_width)
+  display_files.each do |display_file|
+    display_file.each do |file_name|
```

Comment:
```suggestion
  file_grid.each do |files|
    files.each do |file|
```

`display_file`は単数形で配列に見えないので、複数形にしたいですね。
また、mainメソッド内ではfile_nameとは呼んでいないので、プログラム全体でfiles/fileと呼んであげた方がベターです。（もしくはfile_names/file_nameで一貫させる）

Reply:
files/fileで統一しました！

ご確認よろしくお願い致します！